RequestService/RequestService.Repo/Repository.cs

```diff
@@ -197,5 +197,38 @@ public async Task<int> NewHelpRequestAsync(PostNewRequestForHelpRequest postNewR
 
         }
 
+        private async Task<bool> AddJobStatus(int jobID, int userID, HelpMyStreet.Utils.Enums.JobStatuses jobStatus)
```

Comment:
One thing i would change, would be to pass the cancellation token into this function and use it inside _context.SaveChangesASync